x-pack/test/functional/page_objects/rollup_page.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { map as mapAsync } from 'bluebird';
+
+export function RollUpPageProvider({ getService, getPageObjects }) {
+  const testSubjects = getService('testSubjects');
+  const log = getService('log');
+  const PageObjects = getPageObjects(['header', 'common']);
+
+
+  class RollupJobPage {
+    async createNewRollUpJob() {
+      await testSubjects.click('createRollupJobButton');
+    }
+
+    async verifyStepIsActive(stepNumber) {
+      await testSubjects.exists(`createRollupStep${stepNumber}--active`);
+    }
+
+    async addRoleNameandIndexPattern(name, indexPattern) {
+      log.debug(`Adding name ${name} to form`);
+      await testSubjects.setValue('rollUpJobName', name);
+      await testSubjects.setValue('rollUpIndexPattern', indexPattern);
+    }
+
+    async verifyIndexPatternAccepted() {
+      const span = await testSubjects.find('fieldIndexPatternSuccessMessage');
+      const message = await span.findByCssSelector('p');
+      const text = await message.getVisibleText();
+      expect(text).to.be.equal('Success! Index pattern has matching indices.');
+    }
+
+    async setIndexName(name) {
+      await testSubjects.setValue('rollUpIndexName', name);
```

Comment:
for example the test subject and consumption here should be changed to `rollupIndexName`